examples/common/logs_foo_library/foo_library.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+#ifdef ENABLE_LOGS_PREVIEW
+#  include <map>
+#  include <string>
+#  include "opentelemetry/logs/provider.h"
+#  include "opentelemetry/sdk/version/version.h"
+#  include "opentelemetry/trace/provider.h"
+
+namespace logs  = opentelemetry::logs;
+namespace trace = opentelemetry::trace;
+namespace nostd = opentelemetry::nostd;
+
+namespace
+{
+nostd::shared_ptr<trace::Tracer> get_tracer()
+{
+  auto provider = trace::Provider::GetTracerProvider();
+  return provider->GetTracer("foo_library", OPENTELEMETRY_SDK_VERSION);
+}
+
+nostd::shared_ptr<logs::Logger> get_logger()
+{
+  auto provider = logs::Provider::GetLoggerProvider();
+  return provider->GetLogger("foo_library_logger", nostd::span<nostd::string_view>());
```

Comment:
`return provider->GetLogger("foo_library_logger");` will be much simpler ?

Reply:
done